### ISSUE TITLE
Broken links in the README

### ISSUE THREAD

**Issue by cjjdespres** [eclipse-openj9/openj9#17492]
I'm not sure if this is intentional, but in the "Videos and Presentations" section the following two videos are private (at least if you're not signed in to YouTube)

- [JavaOne 2017: John Duimovich and Mike Milinkovich having fun chatting about Eclipse OpenJ9 (and EE4J)](https://www.youtube.com/watch?v=4g9SdVCPlnk)
- [JavaOne 2017: Holly Cummins interviewing Dan Heidinga and Mark Stoodley on Eclipse OpenJ9 and OMR](https://www.youtube.com/watch?v=c1LVXqD3cII)

These three links in the same section redirect to <https://www.ibm.com/support/pages/java-sdk>. I don't know if there's an updated location for them.

- [Reducing Garbage Collection pause times with Concurrent Scavenge and the Guarded Storage Facility](https://developer.ibm.com/javasdk/2017/09/18/reducing-garbage-collection-pause-times-concurrent-scavenge-guarded-storage-facility/)
- [How Concurrent Scavenge using the Guarded Storage Facility Works](https://developer.ibm.com/javasdk/2017/09/25/concurrent-scavenge-using-guarded-storage-facility-works/)
- [Are you still paying for unused memory when your Java app is idle?](https://developer.ibm.com/javasdk/2017/09/25/still-paying-unused-memory-java-app-idle/)

The link [Beginner issues](https://github.com/eclipse-openj9/openj9/issues?q=is%3Aopen+is%3Aissue+label%3Abeginner) at the start of the README seems to refer to an issue tag that's never been used.

**Comment by pshipton** [comment]
@doveye fyi

**Comment by doveye** [comment]
@pshipton sorry, I don't think I can help with this issue. The only involvement our team had with this file was that Sue added some info about Slack, years ago. We didn't have anything to do with the videos or articles (looks like they were added by @DanHeidinga in 2020), so have no idea what happened to those. I went to ibm.developer to see if I could find the articles but it doesn't seem to be working at the moment (it just churns and doesn't show me any content).

Looks like @mstoodle added the "Beginner issues" content at the start of the project - I guess the project can decide if it wants to adopt that tag or just remove this content?

**Comment by mstoodle** [comment]
> Looks like @mstoodle added the "Beginner issues" content at the start of the project - I guess the project can decide if it wants to adopt that tag or just remove this content?

Yeah, we seem to have created "Good first issue" as the expected tag without updating the README link to point at
[https://github.com/eclipse-openj9/openj9/labels/good%20first%20issue](https://github.com/eclipse-openj9/openj9/labels/good%20first%20issue)
